### PR TITLE
Updated the Readme.md file

### PR DESCRIPTION
**Pull request by Big-Zude**

### CODE REVIEW

**Comment by Big-Zude** [comment]
thanks to you for the opportunity.

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ read the [code of conduct](https://github.com/devconzm/devconzm/blob/master/CODE
 - git
 
 clone the repo
```
Comment (OlivierJM) [inline README.md]: add double space at the end of repo so that both lines are separated

**Comment by OlivierJM** [comment]
#16

**Comment by OlivierJM** [comment]
@allcontributors please add @Big-Zude for docs

File: README.md (hunk 1 above)
Comment (Big-Zude) [reply README.md]: alright

**Comment by Big-Zude** [comment]
is everything ok?

**Comment by OlivierJM** [comment]
Yeah everything is okay, I have merged it, you can check the changes in the main Readme .
Thanks

**Pull request merged**